<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录取卡号</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="modify">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modfyback">
        <property name="text">
         <string>撤销修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insert">
        <property name="text">
         <string>添加记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_2">
        <property name="text">
         <string>删除选中行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectall">
        <property name="text">
         <string>显示全表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="orderbyasc">
        <property name="text">
         <string>按id升序排序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="orderbydesc">
        <property name="text">
         <string>按id降序排序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="serialPortComboBox"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>卡号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cardLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>串口状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trafficLabel">
        <property name="text">
         <string>发送接收</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_Analog"/>
    <addaction name="action_Send"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Analog">
   <property name="text">
    <string>模拟串口</string>
   </property>
  </action>
  <action name="action_Send">
   <property name="text">
    <string>发送数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
